docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+networks:
+  intergration-test:
```

Comment:
"integration-tests" ?

Reply:
solved here https://github.com/geosolutions-it/docker-geoserver/pull/43/commits/dd8970b203ef4e7bee74c4db00dea8cc284f4072